crates/bin/pcli/src/command/threshold.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::Result;
+
+use crate::{terminal::ActualTerminal, App};
+
+#[derive(Debug, clap::Subcommand)]
+pub enum ThresholdCmd {
+    /// Follow along with the threshold signing process
+    Follow,
```

Comment:
Should this be `sign`? `pcli threshold sign` seems more intuitive than `pcli threshold follow` to me,  but this is a bit of a bikeshed to be sure.